.github/workflows/ci.yaml

```diff
@@ -55,6 +56,7 @@ jobs:
         # Disable clang-tidy for ikfast plugins as we cannot fix the generated code
         find $BASEDIR/target_ws/build -iwholename "*_ikfast_plugin/compile_commands.json" -exec rm {} \;
       BUILDER: ${{ matrix.env.BUILDER || 'catkin_tools' }}
+      DOCKER_RUN_OPTS: ${{ matrix.env.IKFAST_TEST && '-v /var/run/docker.sock:/var/run/docker.sock' }}
```

Comment:
for my own education: why do you need to forward that socket here? My guess would be that test_ikfast_plugin needs to create a docker container from within docker (now that you moved the generation into the container), but I'm not certain and a comment might be appropriate :-)

Reply:
Your guess is correct. I need to run docker from within the industrial_ci docker container. Will add a comment.